pkg/util/yaml/decoder.go

```diff
@@ -30,6 +30,10 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	MaxScanTokenSize = 512 * 1024
```

Comment:
Our YAML could be up to ~1MB for secrets and config (which are the most likely to cause problems here).  

I would prefer to fix this the right way, since we don't need to be allocating megabytes of memory just to scan a simple file.  In this case I think the more complex fix is the right approach.